R/outpack_tools.R

```diff
@@ -222,6 +226,9 @@ orderly_metadata_extract <- function(expr = NULL, name = NULL, location = NULL,
 
   envir <- environment()
   ret <- data_frame(id = ids)
+  if (isTRUE(allow_remote) || length(location) > 0) {
+    ret$local <- ids %in% root$index$unpacked()
+  }
```

Comment:
local boolean is good, but do we not have any way of just having a "location" column that has "local" as a special value and the name of any other location someone has specified? i can imagine users potentially wanting info about what remote location a packet has come from

Reply:
agree that this would be nicer....*but* as packets can be found at multiple locations it's awkward as we have to return a list column. If we think that's worth it it's very easy to add, but they're a bit of a pain to use imo (we do return them liberally elsewhere in this function though)

Reply:
OK, I think this is worthwhile and have done it